test/integration/krane_deploy_test.rb

```diff
@@ -537,17 +532,6 @@ def test_wait_false_ignores_non_priority_resource_failures
     ], in_order: true)
   end
 
-  def test_extra_bindings_should_be_rendered
```

Comment:
Was this removed because this is already covered by `test_dynamic_erb_collection_works`?

Reply:
Yep 👍